.github/ISSUE_TEMPLATE.md

```diff
@@ -14,18 +14,16 @@
 #### Browser Information
 <!-- Describe your workspace in which you are having issues-->
 
-* Browser Name, Version: 
-* Operating System: 
-* Mobile, Desktop, or Tablet: 
+* Browser Name, Version:
+* Operating System:
+* Mobile, Desktop, or Tablet:
 
 #### Your Code
 <!-- If relevant, paste all of your challenge code in here -->
 ```js
- 
- 
- 
+
+
+
```

Comment:
@Greenheart oddly enough, I did not make a change here. I've noticed this recently in my PRs, something is automatically cleaning up white space.